comps/embeddings/multimodal_embeddings/multimodal_langchain/local_mm_embedding.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from langsmith import traceable  # type: ignore
+
+from comps import (
+    CustomLogger,
+    EmbedDoc,
+    EmbedMultimodalDoc,
+    MultimodalDoc,
+    ServiceType,
+    TextDoc,
+    TextImageDoc,
+    opea_microservices,
+    register_microservice,
+)
+from comps.embeddings.multimodal_embeddings.bridgetower import BridgeTowerEmbedding
+
+logger = CustomLogger("local_multimodal_embedding")
+logflag = os.getenv("LOGFLAG", False)
+
+port = int(os.getenv("MM_EMBEDDING_PORT_MICROSERVICE", 6600))
+
+
+@register_microservice(
+    name="opea_service@local_multimodal_embedding",
+    service_type=ServiceType.EMBEDDING,
+    endpoint="/v1/embeddings",
+    host="0.0.0.0",
+    port=port,
+    input_datatype=MultimodalDoc,
+    output_datatype=EmbedMultimodalDoc,
+)
+@traceable(run_type="embedding")
```

Comment:
please remove langsmith related code.

Reply:
Thank for your feedback. We have removed langsmith here.